docs/ref/clvm.md

```diff
@@ -633,31 +757,36 @@ A transaction block is considered "full" when it contains 2000 outputs. For this
 
   > Note: A transaction with only one input and one output is also possible. In theory, a block could therefore hold up to 2000 transactions, in which case the network would process an average of 38.5 (2000/51.95) transactions per second.
 
-With this goal in mind, Chia has created a generator program that processes 2000 compressed inputs and outputs. This program simulates a "full block".
+With this goal in mind, Chia has created a **generator program** that processes 2000 compressed inputs and outputs. This program simulates a "full block".
 
 To calculate the total amount of time for a Raspberry Pi 4 to process a full block, we must take into account three factors:
-* The time required to run the generator program (2000 inputs and outputs): 5.2 seconds
-* The time required to validate 2000 public keys: 2.2 seconds
-* The time required to validate 2000 aggregate signatures: 10.63 seconds
+* The time required to run the generator program (2000 inputs and outputs)
+  * The Raspberry Pi 4 accomplishes this in 5.2 seconds
+* The time required to validate 2000 public keys
+  * 2.2 seconds
+* The time required to validate 2000 aggregate signatures
+  * 10.63 seconds
 
-Therefore, the total amount of time required for a Raspbery Pi 4 to process a full block is 5.2 + 2.2 + 10.63 = 18.03 seconds. This is 10.095 seconds faster than the minimum time between blocks, and 33.92 seconds faster than the average. When considering other factors such as network latency and time required to fetch a full proof ([640 ms on a slow HDD](https://docs.chia.net/docs/03consensus/proof-of-space#farming)), this leaves plenty of leeway for a Raspberry Pi 4 to stay synced and collect farming rewards.
+Therefore, the total amount of time required for a Raspberry Pi 4 to process a full block is 5.2 + 2.2 + 10.63 = 18.03 seconds. This is 10.095 seconds faster than the minimum time between blocks, and 33.92 seconds faster than the average. When considering other factors such as network latency and time required to fetch a full proof ([640 ms on a slow HDD](https://docs.chia.net/docs/03consensus/proof-of-space#farming)), this still allows plenty of leeway for a Raspberry Pi 4 to stay synced and collect farming rewards.
 
-## Can a Raspberry Pi 4 sync from genesis?
+## Maximum cost per block rationale
```

Comment:
this title change doesn't look right. Its content still talk about an RPi staying synced.

Oh, I see, this is the higher-level title, and the ones below are subtitles. It might make sense to mention that we use the RPi 4 as our benchmark for what performance we expect from farmers.